docs/vtex-io/Perks/best-practices/vtex-io-documentation-best-practices-for-optimizing-performance.md

```diff
@@ -6,171 +6,171 @@ createdAt: "2020-11-11T12:55:53.602Z"
 
-As a result, we recommend that stores already using the new `minicart.v2` and `add-to-cart-button` blocks (instead of the `minicart` and `buy-button`, respectively) set aside the `OrderFormProvider` legacy. This way, consuming a single provider, you'll notice improvements in your store performance.
+As a result, we recommend that the stores that already use the new `minicart.v2` and `add-to-cart-button` blocks (instead of `minicart` and `buy-button`, respectively) discontinue `OrderFormProvider` legacy. By consuming from a single provider, you will notice performance improvements in your store.
 
-> ℹ️ More details on how to perform this optimization can be found in the documentation on [Enabling OrderForm optimization](https://developers.vtex.com/docs/guides/vtex-io-documentation-enabling-order-form-optimization/).
+> ℹ️ More details on how to implement this optimization can be found in the documentation on [Enabling OrderForm optimization](https://developers.vtex.com/docs/guides/vtex-io-documentation-enabling-order-form-optimization/).
 
 ### Deactivating the VTEX IO native service worker
 
-The VTEX IO platform provides a native service worker to every store using the Store Framework. However, since a website can only run one service worker at a time, you may face conflicts when working with a third-party service that provides its own service worker.
+The VTEX IO platform provides a native service worker to every store that uses Store Framework. However, since a website can only run one service worker at a time, you may face conflicts when working with a third-party service that provides its own service worker.
 
-Hence, you can choose to deactivate the native service worker provided by VTEX IO in order to successfully use a third-party solution in your store.
+You can deactivate the native service worker provided by VTEX IO to successfully use a third-party solution in your store.
 
-> ℹ️ For more information, check our documentation on [Deactivating the VTEX IO native service worker](https://developers.vtex.com/docs/guides/vtex-io-documentation-deactivating-the-vtex-io-native-service-worker/).
+> ℹ️ For more information, read our documentation on [Deactivating the VTEX IO native service worker](https://developers.vtex.com/docs/guides/vtex-io-documentation-deactivating-the-vtex-io-native-service-worker/).
 
 ### Optimizing critical CSS
 
-The content which a user first sees when opening a web page is known as *above-the-fold*. Besides, since this part of the page must be loaded quickly for better user experience, it can also be referred to as *critical*.
+The content which a user first sees when opening a web page is known as *above-the-fold* content. This part of the page can also be called *critical* because it needs to be loaded quickly to provide a better user experience.
```

Comment:
```suggestion
The content which a user first sees when opening a web page is known as above-the-fold content. This part of the page can also be called critical because it needs to be loaded quickly to provide a better user experience.
```